### ISSUE TITLE
Switch invoker to authorizedParty

### ISSUE THREAD

**Issue by cwebber** [w3c-ccg/zcap-spec#20]
`invoker` is a bit confusing because it makes the capability document look like an invocation when you're reading it.  @dmitrizagidulin suggests `authorizedParty` which is closer to the oauth group does

**Comment by danfinlay** [comment]
I think “recipient” also works pretty well.

**Comment by dlongley** [comment]
We want to be able to distinguish `invoker` from `delegator`, so if we change the name, we should change both, perhaps to: `authorizedInvoker` and `authorizedDelegator`. I'm also not sure how confusing it is ... and whether or not a change is really necessary must now be weighed against the cost of changing what is already being deployed today.

**Comment by danfinlay** [comment]
I agree with not breaking existing things, just riffing on the wording because having just written up the type, `invoker` stood out as a bit awkward :)

**Comment by dlongley** [comment]
We are likely going to change `invoker` to `authorizedInvoker` and `delegator` to `authorizedDelegator` to avoid confusion. The reason for the change is that specifying who delegated a capability via the `delegator` property is useful -- and this would be in conflict with using it to mean who is authorized to further delegate. Since we'd be changing to use `authorizedDelegator` it would make sense to align `invoker` by changing it to `authorizedInvoker` even though we don't have a use case for `invoker` at this time. We will likely make this change in our ocapld.js implementation very shortly.

cc: @cwebber

**Comment by dlongley** [comment]
We're going to experiment a bit more with our implementation per the last comment -- it may turn out to be more confusing in some ways to use `authorizedDelegator` than just `delegator` -- based on the fact that you need to check `parentCapability.authorizedDelegator` when verifying, not `capability.authorizedDelegator`.

**Comment by gobengo** [comment]
In the spec, `invoker` was changed by @dmitrizagidulin to `controller` in https://github.com/w3c-ccg/zcap-spec/commit/e2e5ff3cf8dc823a3daa422b9a9fe171c3dc6d7d .
This is consistent with [did-core](https://www.w3.org/TR/did-1.0/#did-controller) and [cid-v1.0](https://www.w3.org/TR/cid-1.0/#controllers)
It had been discussed here and seems based on implementation experience. https://github.com/w3c-ccg/zcap-spec/issues/39#issuecomment-1064189328

`controller` in general is a verification relationship that is super broad, i.e. the verification methods authorized that way can usually do anything.

> We want to be able to distinguish invoker from delegator

This is good too. When that's important, the `capabilityDelegation` and `capabilityInvocation` properties / verification relationships seem appropriate. The former is already mentioned in zcap-spec.

@cwebber given the above, it seems like this issue has been resolved to some extent. `invoker` is no longer in zcap-spec. wdyt of closing it?
